images_config/ks-robot-master.cfg

```diff
@@ -271,11 +271,15 @@ chmod +x /usr/local/bin/socket_buffer.py
 #initctl start candump_any
 %end
 
-################ INSTALL NOMACHINE ################ 
+################ INSTALL PULSEAUDIO ################ 
 %post --erroronfail --interpreter /bin/bash
 apt-get install pulseaudio -y
-NOMACHINE_VERSION=5.3.12_10
-wget -O /root/nomachine_${NOMACHINE_VERSION}_amd64.deb http://download.nomachine.com/download/5.3/Linux/nomachine_${NOMACHINE_VERSION}_amd64.deb
+%end
+
+################ INSTALL NOMACHINE ################ 
+%post --erroronfail --interpreter /bin/bash
+NOMACHINE_VERSION=6.0.62_6
+wget -O /root/nomachine_${NOMACHINE_VERSION}_amd64.deb http://download.nomachine.com/download/6.0/Linux/nomachine_${NOMACHINE_VERSION}_amd64.deb
 dpkg -i /root/nomachine_${NOMACHINE_VERSION}_amd64.deb
```

Comment:
maybe the same can be done for `nomachine`, too
like so `nomachine=NOMACHINE_VERSION` (see https://askubuntu.com/a/428778)?

Reply:
the problem is that nomachine is not available through the ubuntu package repository. so we need to manually download the deb file and install it.
Obviously nomachine does not provide older version on their servers any more, so we have to update to a more recent version manually